src/test/java/io/lettuce/core/pubsub/PubSubCommandTest.java

```diff
@@ -60,7 +65,8 @@
  * @author Tihomir Mateev
  * @author Ali Takavci
  */
-class PubSubCommandTest extends AbstractRedisClientTest implements RedisPubSubListener<String, String> {
+class PubSubCommandTest extends AbstractRedisClientTest
+        implements RedisPubSubListener<String, String> {
```

Comment:
Going forward, it would be good to create an inner class that implements `RedisPubSubListener` for a cleaner design.

Reply:
done